src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import core.basesyntax.enums.Color;
+import core.basesyntax.enums.FigureType;
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final Random RANDOM = new Random();
```

Comment:
it's better to make Random a simple field, not a constant. You'll figure out the reason of it in the further modules/howeworks. The same for color supplier